control/__init__.py

```diff
@@ -101,6 +101,7 @@
 from .config import *
 from .sisotool import *
 from .passivity import *
+from .sysnorm import *
```

Comment:
Please add an `__all__` to sysnorm.py in order not to reference `np`, `la`, and `ct` as `control.sysnorm.np` and so on.

Reply:
Thanks @bnavigator , done.